netebpfext/net_ebpf_ext_program_info.h

```diff
@@ -88,14 +88,17 @@ static const ebpf_helper_function_prototype_t _sock_addr_ebpf_extension_helper_f
      BPF_FUNC_sock_addr_set_redirect_context,
      "bpf_sock_addr_set_redirect_context",
      EBPF_RETURN_TYPE_INTEGER,
-     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_PTR_TO_READABLE_MEM, EBPF_ARGUMENT_TYPE_CONST_SIZE}}};
+     {EBPF_ARGUMENT_TYPE_PTR_TO_CTX, EBPF_ARGUMENT_TYPE_PTR_TO_READABLE_MEM, EBPF_ARGUMENT_TYPE_CONST_SIZE}},
+    };
 
 enum _sock_addr_global_helper_functions
 {
     SOCK_ADDR_GLOBAL_HELPER_GET_CURRENT_PID_TGID,
     SOCK_ADDR_GLOBAL_HELPER_GET_CURRENT_LOGON_ID,
     SOCK_ADDR_GLOBAL_HELPER_IS_CURRENT_ADMIN,
     SOCK_ADDR_GLOBAL_HELPER_GET_SOCKET_COOKIE,
+    SOCK_ADDR_GLOBAL_HELPER_GET_PROCESS_START_KEY,
```

Comment:
since these are now global helper functions, we dont need to include these here.

This list is only for the global helper functions that sock_addr extension wants to overwrite with its implementation, which is not the case here.